webpage/jupyter_notebook_jekyll_theme/_sass/_navigation.scss

```diff
@@ -175,7 +175,7 @@
 .greedy-nav {
   position: relative;
   min-width: 250px;
-  background-color: $greedy-nav-background-color;
+  background-color: #5E5C57;
```

Comment:
I kind of wanted to preserve the original variables, but with git that does not really seem to be relevant...